src/static/icons/github-handler.svg

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Where does this file come from?

Reply:
I took it from https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/white/svg/github.svg then resized it with Inkscape. I didn't realize Inkscape were adding all this metadata, I'm going to edit the file to strip it.

Reply:
As we're embedding Font Awesome, could we use it directly?

Reply:
I think the embedded font-awesome doesn't include svg files. I wanted to mimic the way Twitter and website links are written, with a svg file set as background.

I can make a change to use the font version of the icon.

Reply:
@Betree I'm not sure in the end between `font-awesome` and svg. Because we have font awesome available, it's great to have one image less to maintain. Maybe we could also then migrate the twitter one. But, for this PR, sounds more straightforward to mimic what's already there. So, your call.

Reply:
I'm not opinionated on the SVG vs Font debate as both approaches have their up/downside but I do agree with not having to maintain images files and using the existing `font-awesome` library for consistency.

Migrating the (few) existing `svg` icons this way would be nice. May I create an issue for that?

Reply:
Ok, we can keep it like this with svg, then follow up with font-awesome in a separate issue.